emukit/examples/profet/meta_benchmarks/meta_svm.py

```diff
@@ -0,0 +1,85 @@
+
+import pickle
+from functools import partial
+from typing import Tuple
+
+import numpy as np
+import torch
+from emukit.core import ContinuousParameter, ParameterSpace
+from emukit.core.loop.user_function import UserFunctionWrapper
+from emukit.examples.profet.meta_benchmarks.architecture import get_default_architecture_classification, \
+    get_default_architecture_cost
+
+
+def meta_svm(fname_objective: str, fname_cost: str, noise: bool = True) -> Tuple[UserFunctionWrapper, ParameterSpace]:
+    """
+    Interface to the Meta-SVM benchmark which imitates the hyperparameter optimization of a
+    support vector machine on OpenML like classification datasets.
+    Offline generated function samples can be download here:
+
+    http://www.ml4aad.org/wp-content/uploads/2019/05/profet_data.tar.gz
+
+    NOTE: make sure that the index for the objective function and the cost function match,
+    e.g for sample_objective_i.pkl  and sample_cost.pkl the index i should be the same.
```

Comment:
`sample_cost_i`? again
```suggestion
    e.g for sample_objective_i.pkl  and sample_cost_i.pkl the index i should be the same.
```

Reply:
fixed